kernel/README.md

```diff
@@ -0,0 +1,6 @@
+
+## ewasm-kernel classic
```

Comment:
I would definitely not import here but rather into its own repo where it belongs to (it was at ewasm/ewasm-kernel)

Reply:
I was thinking it's a more convenient workflow to modify files in a subdirectory, since testing work-in-progress across separate modules can be a hassle (have to delete it from node_modules and reinstall for every change).

But I'm happy to break it off back into ewasm-kernel.

Reply:
> have to delete it from node_modules and reinstall for every change

try [`npm link`](https://docs.npmjs.com/cli/link) to do that. Also what commit did this come from?

Reply:
good tip.  ewasm-kernel is forked from https://github.com/primea/js-primea-hypervisor/tree/26787afda66ed5f43ebf8e40d4edefbbcc133e01/

Reply:
https://github.com/ewasm/ewasm-kernel